### ISSUE TITLE
Problem upgrading to 0.9.3

### ISSUE THREAD

**Issue by dajare** [shaarli/Shaarli#1052]
I am attempting to upgrade from 0.9.2 to 0.9.3. At first, I just replaced the `index.php` file, since it seemed that this was all that was needed. However, that left me with a white screen when I visited the site, so I replaced it with my backed-up `index.php` (0.9.2) version until I had time to do a full, proper re-install.

So this morning I did a "full" 0.9.2 > 0.9.3 upgrade:

- I backed up the full Shaarli directory (including `/data` directory, of course!).
- I uploaded the 0.9.3 zip to my host, unzipped, and put the contents in my Shaarli directory.
- I replaced the `/data` directory with my backed-up version.
- And ... whitescreen! :(

I checked page source and there's nothing. This is hard for me to work out! I checked the contents of `.htaccess` files, and all seems fine. The permissions are as they should be: `755` for directories; `644` for files. My PHP version is 5.6 ("[supported][p]").

I can't think of what else to check. Any help appreciated!

[p]: https://shaarli.readthedocs.io/en/master/Server-requirements/

**Comment by ArthurHoaro** [comment]
A blank page is usually a PHP fatal error. Can you check your web server logs?

**Comment by virtualtam** [comment]
> The permissions are as they should be: 755 for directories; 644 for files.

Not necessarily. What is needed is that the user running the web server (Apache HTTPD, Nginx, PHP-FPM) has read access to PHP resources, and write access to data and cache folder, see https://shaarli.readthedocs.io/en/master/Shaarli-configuration/

See also https://shaarli.readthedocs.io/en/master/Upgrade-and-migration/ for the recommended upgrade paths.

**Comment by dajare** [comment]
Thanks both. I actually did read the docs (!) but was too careless. 

I did the "fresh installation" version ... but because I moved my `/data` directory in *before* running the configuration, I got the whitescreen. I removed my `/data`, ran the install script, *then* restored my `/data` directory, and now all is well.

(Actually, the Material theme which I use has a glitch, but that's not a problem for here!)

*Many* thanks for the speedy replies. Yet another case of "user error", I'm afraid.  :flushed:
